src/worker/core/GameSim.basketball/index.ts

```diff
@@ -309,6 +312,7 @@ class GameSim {
 		this.elamActive = false;
 		this.elamDone = false;
 		this.elamTarget = 0;
+		this.isOneTwos = isOneTwos;
```

Comment:
Add a new setting to `defaultGameAttributes.ts` and then you can just access it here with `g.get("isOnesTwos")`, similar to other settings used in this file like `numPlayersOnCourt`. Also in `defaultGameAttributes.ts` add the new setting to `gameAttributesKeysSportSpecific` (might require merging with master) so it's not included in other sports exports.